File: Driver-App/app/src/main/java/com/tkpmnc/sgtaxidriver/home/interfaces/ServiceListener.java
package com.tkpmnc.sgtaxidriver.home.interfaces;

import com.tkpmnc.sgtaxidriver.common.model.JsonResponse;


public interface ServiceListener {

    void onSuccess(JsonResponse jsonResp, String data);

    void onFailure(JsonResponse jsonResp, String data);


    /*void onSuccessResponse(JsonResponse jsonResp, String data);*/
}

File: User-App/app/src/main/java/com/tkpmnc/sgtaxiuser/appcommon/interfaces/ServiceListener.kt
package com.tkpmnc.sgtaxiuser.appcommon.interfaces

/**
 * @package com.tkpmnc.sgtaxiuser
 * @subpackage interfaces
 * @category ServiceListener
 * 
 * 
 */

import com.tkpmnc.sgtaxiuser.appcommon.datamodels.JsonResponse

/*****************************************************************
 * ServiceListener
 */
interface ServiceListener {

    fun onSuccess(jsonResp: JsonResponse, data: String)

    fun onFailure(jsonResp: JsonResponse, data: String)
}
